### PR TITLE
Update Hazelcast version in README examples

### PR DESCRIPTION
**Pull request by Holmistr**

### CODE REVIEW

**Review by vladoschreiner — APPROVED** [review]

**Review by huseyinbabal — CHANGES_REQUESTED** [review]
Let's wait for Hazelcast Cloud 3.4.0, since you cannot just set 5.0 in your client. And this pr depends on https://github.com/hazelcast/hazelcast-cloud-cli/pull/30
Screen Shot 2021-12-23 at 13 12 09

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ hzcloud starter-cluster create \
   --name=mycluster \
   --region=us-west-2 \
   --total-memory=0.2 \
-  --hazelcast-version=4.0
+  --hazelcast-version=5.0
```
Comment (huseyinbabal) [inline README.md]: The current version is 5.0.2-1, so better to have `--hazelcast-version=5.0.2-1`. In the future, we will improve hz version selection smarter on backend side.

File: README.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ hzcloud enterprise-cluster create \
   --cloud-provider=aws \
   --region=eu-west-2 \
   --zones=eu-west-2b \
-  --hazelcast-version=4.0 \
+  --hazelcast-version=5.0 \
```
Comment (huseyinbabal) [inline README.md]: The current version is `5.0.2`, so better to have `--hazelcast-version=5.0.2`. In the future, we will improve hz version selection smarter on backend side.

File: README.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ hzcloud starter-cluster create \
   --name=mycluster \
   --region=us-west-2 \
   --total-memory=0.2 \
-  --hazelcast-version=4.0
+  --hazelcast-version=5.0.2
```
Comment (huseyinbabal) [inline README.md]: This should be `5.0.2-1`

Comment (Holmistr) [reply README.md]: Fixed. 

Just a note for anybody looking - this doesn't look very user friendly, adding a version that practically doesn't exist publicly. But Huseyin acknowledged that and we'll create a task for improving this on the backend side. I'll add a link to it here once created.

**Review by huseyinbabal — APPROVED** [review]

**Pull request merged**